MODULE.bazel

```diff
@@ -0,0 +1,40 @@
+module(
+    name = "hessian2-codec",
+    repo_name = "com_alibaba_hessian2_codec",
+)
+
+bazel_dep(
+    name = "bazel_skylib",
+    version = "1.4.1",
+)
+bazel_dep(
+    name = "abseil-cpp",
+    version = "20220623.1",
+    repo_name = "com_google_absl",
+)
+bazel_dep(
+    name = "fmt",
+    version = "8.1.1",
+    repo_name = "com_github_fmtlib_fmt",
+)
+bazel_dep(
+    name = "googletest",
+    version = "1.11.0",
+    repo_name = "com_google_googletest",
+)
+bazel_dep(
+    name = "bazel-compile-commands-extractor",
```

Comment:
If the name of bazel_dep does not match the definition in the dependent Module, Bazel will throw an error.